clang/test/Preprocessor/cuda-macos-includes.cu

```diff
@@ -1,13 +1,6 @@
-// RUN: %clang -cc1 -fcuda-is-device -isysroot /var/empty \
-// RUN:   -triple nvptx-nvidia-cuda -aux-triple i386-apple-macosx \
-// RUN:   -E -fcuda-is-device -v -o /dev/null -x cuda %s 2>&1 | FileCheck %s
-
-// RUN: %clang -cc1 -isysroot /var/empty \
-// RUN:   -triple i386-apple-macosx -aux-triple nvptx-nvidia-cuda \
-// RUN:   -E -fcuda-is-device -v -o /dev/null -x cuda %s 2>&1 | FileCheck %s
-
 // Check that when we do CUDA host and device compiles on MacOS, we check for
 // includes in /System/Library/Frameworks and /Library/Frameworks.
 
-// CHECK-DAG: ignoring nonexistent directory "/var/empty/System/Library/Frameworks"
-// CHECK-DAG: ignoring nonexistent directory "/var/empty/Library/Frameworks"
+// RUN: %clang -isysroot /var/empty -target unknown-nvidia-cuda -v -fsyntax-only -x cuda %s -### 2>&1 | FileCheck %s
```

Comment:
@jhuber6 Pinging you for Cuda questions, feel free to let me know if I should be asking someone else instead.

This test doesn't pass with the current patch. It looks like the Cuda driver isn't calling the Darwin toolchain driver when setting up search paths. Previously this was all done in the frontend so that wasn't a problem, but I don't understand how that's supposed to work. Do you have familiarity with this part of Clang?

Reply:
@Artem-B Does NVIDIA even support CUDA on MAC anymore?

Reply:
Nope. I believe they've dropped MacOS in CUDA-11. 10.2 was the last one.
https://docs.nvidia.com/cuda/archive/10.2/cuda-toolkit-release-notes/index.html#cuda-general-new-features

```
CUDA 10.2 (Toolkit and NVIDIA driver) is the last release to support macOS for developing and running CUDA applications. Support for macOS will not be available starting with the next release of CUDA.
```

CUDA on MAC has never been properly supported. It kind-of-sort-of worked, but nobody owned or maintained it beyond whatever happened to be touched by LLVM tests.
By now, I think we can remove cuda-on-mac tests and related code.

Reply:
Oh, okay then! So I'll just remove that test entirely.